### DISCUSSION TITLE
Logsender

### DISCUSSION THREAD

**Discussion by AmanRajAryan** [AndroidIDEOfficial/AndroidIDE#1873] [General]
Please can anybody let me how to use logsender , I've created a projected and simply on the click of a button I've added `Log.d("test" , "log");`

**Comment by CyberXCoder-6649** [comment]
Swipe from down to open terminal and then goto App logs section and find your logs there

**Reply by AmanRajAryan** [reply]
Logs aren't showing up 😔

**Reply by CyberXCoder-6649** [reply]
Restart IDE and run app
